src/pages/guides/quickstart/index.md

```diff
@@ -15,43 +15,58 @@ keywords:
   - Getting started
   - Browser support
 title: Quickstart Guide
-description: This is the quickstart guide for Adobe Express Embed SDK
+description: This is the Quickstart Guide for Adobe Express Embed SDK
 contributors:
   - https://github.com/amandahuarng
   - https://github.com/nimithajalal
+  - https://github.com/undavide
 ---
 # Quickstart Guide
 
-This guide explains how to start using the Adobe Express Embed SDK in your own application.
+In this guide, you will learn how to get started with the Adobe Express Embed SDK.
 
 ## Step 1: Get an API Key
 
-Follow the below steps to get an API Key.
+API keys—one for each project—are required to use the Adobe Express Embed SDK. You can get one logging to your [Adobe Developer Console](https://developer.adobe.com/console); if you need a refresher on the Console, you can read more about it [here](https://developer.adobe.com/developer-console/docs/guides/getting-started/).
 
-1. Navigate to the [Developer Console](https://developer.adobe.com/console).
-2. Select **APIs and services**.
-3. Search for **Adobe Express Embed SDK**.
-4. Click **Create project**.
+![](../images/quickstart_console.png)
 
-On the **Configure API** section,
+Select **APIs and services** from the top navigation bar; you'll see there a list of the available ones. Find the **Adobe Express Embed SDK** card and click the **Create project** button.
 
-1. Add your **Allowed domains** in the fields. To prevent unauthorized use of your client ID on another website, restrict usage to the domains you authorize.
-2. Click **Save configured API**. The API key is displayed on the screen.
+![](../images/quickstart_apis-and-services.png)
+
+On the **Configure API** section, you must insert up to five domains that will be allowed to use your API key. This is a security measure to prevent unauthorized use of your client ID on another website, restricting usage to the domains you have authorized.
+
+![](../images/quickstart_configure-domain.png)
+
+For local development, you can use `localhost` specifying the port with the colon `:` syntax—in the screenshot above, it's 5555. Please note that all connections must be secure (HTTPS) for both production and development environments.
+
+Once you have added all the domains, click **Save configured API**. The API key will be displayed on the screen.
+
+![](../images/quickstart_complete.png)
+
+Copy the API key and save it in a secure location. You will need it to initialize the SDK in your project. The API cannot be modified once it's created, but you can always edit the list of domains. Click on the **API key** link under the **Credentials** section on the left column:
+
+![](../images/quickstart_api.png)
+
+There's an **Edit credential** button on the top right corner of the screen. Click on it to edit the list of domains. **Save** your changes when you're done.
+
+![](../images/quickstart_edit-credentials.png)
 
 ## Step 2: Embed the SDK
```

Comment:
Are there three ways of Embedding the SDK? If yes, lets add this in the beginning of this section.

- [Load via script tag](http://localhost:8000/guides/quickstart/#load-via-script-tag)
- [Using import](http://localhost:8000/guides/quickstart/#using-import)
- [Dynamic script](http://localhost:8000/guides/quickstart/#dynamic-script)